css/middle_details_style.css

```diff
@@ -82,3 +93,18 @@ footer {
 #box1, #box2, #box3 {
   height: 100%;
 }
+.go-down {
+  height: 0%;
+}
+.go-full {
+  height: 100%;
+}
+.go-to-20 {
+  height: 20%;
+}
+.go-to-60 {
```

Comment:
made the animation for the middle section as well,  which gives it a box shadow when selected